### PR TITLE
Modify rule S5549: LaYC format

### PR DESCRIPTION
**Pull request by joke1196**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

**Review by guillaume-dequenne-sonarsource — APPROVED** [review]
LGTM.

File: rules/S5549/python/rule.adoc [hunk 1]
```diff
@@ -1,16 +1,19 @@
+This rule raises an issue when a function is called with multiple values for the same parameter.
+
 == Why is this an issue?
 
-When a function is called, it accepts only one value per parameter. Python interpreters will raise a SyntaxError when they see something like ``++myfunction(a=1, a=2)++``, but there are other cases which will only fail at runtime:
+When a function is called, it accepts only one value per parameter. The Python interpreter will raise a `SyntaxError` when the same parameter is provided more than once, i.e. `myfunction(a=1, a=2)`.
 
-* An argument is provided by value and position at the same time.
-* Some arguments are provided via unpacking and the same argument is provided twice.
+Other less obvious cases will also fail at runtime, when:
```
Comment (guillaume-dequenne-sonarsource) [inline rules/S5549/python/rule.adoc:7]: Those will fail with a `TypeError` rather than a `SyntaxError`. Not sure if that should be mentioned.

Comment (jean-jimbo-sonarsource) [reply rules/S5549/python/rule.adoc:7]: For clarity I think it should be mentioned.

**Review by jean-jimbo-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**